app/js/components/ReactReduxSelect.jsx

```diff
@@ -0,0 +1,36 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016-2017, QIIME 2 development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
+
+import React from 'react';
+import Select from 'react-select-plus';
+import style from '!style-loader!css-loader!react-select-plus/dist/react-select-plus.css'; // eslint-disable-line
+
+
+class ReactReduxSelect extends React.Component {
```

Comment:
My impression was that `react-select` simply requires you to fill in what to do when a value is selected (this let's you middle-ware things a bit). Redux doesn't reset anything, it's just kind of the expectation of this library that you would map `onChange` to the react-component's `state` (or something else).

If it didn't allow you to set the value yourself, then it wouldn't be possible to intercept the state of this component for `redux` (which we aren't super interested in anyways).